display_list/display_list_unittests.cc

```diff
@@ -1377,6 +1377,18 @@ TEST(DisplayList, DisplayListFullPerspectiveTransformHandling) {
   }
 }
 
+TEST(DisplayList, DisplayListTransformResetHandling) {
```

Comment:
Good to know. I just followed the pattern of the test right above. I can update in a later patch.

Reply:
File it as a follow-on Issue. We have better fish to fry for now.

Reply:
So, I couldn't really find the method you mentioned. Can you link it here?

Reply:
This section, but it is only implemented for SkPaint-kinds of attributes...

https://github.com/flutter/engine/blob/5a80834bd55979647e39e7724c554003507d1194/display_list/display_list_builder.h#L113

Reply:
We should have getMatrix/M44, but Builder doesn't currently keep a matrix filled out. Also getClipBounds, but the same issue that it doesn't maintain one.

I'm planning to integrate some of the utils into the builder (bounds, for example) and these do track the matrix and clip, so maybe this isn't doable until some of that work is done. Alternatively we could just track the matrix and clip bounds in Builder without integrating the utils.

The attributes were tracked anyway to avoid emitting duplicates, but there was no reason to track the matrix or clip (yet).

Either way, it's more than just adding the getter methods.

Reply:
Filed https://github.com/flutter/flutter/issues/100173 for this.